inst/rmarkdown/templates/ims/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,422 @@
+The following is an example of an *enumerated* list, two levels deep.
+\begin{longlist}
+\item[1.]
+This is the first item of an enumerated list.  Each item
+in the list is marked with a ``tick.''.  The document
+style determines what kind of tick mark is used.
+\item[2.]
+This is the second item of the list.  It contains another
+list nested inside of it.
+\begin{longlist}
+\item
+This is the first item of an enumerated list that
+is nested within.  
+\item
+This is the second item of the inner list.  \LaTeX\
+allows you to nest lists deeper than you really should.
+\end{longlist}
+This is the rest of the second item of the outer list.
+\item[3.]
+This is the third item of the list.
+\end{longlist}
+
+## Punctuation
+
+Dashes come in three sizes: a hyphen, an intra-word dash like "$U$-statistics" or "the time-homogeneous model";
+a medium dash (also called an "en-dash") for number ranges or between two equal entities like "1--2" or "Cauchy--Schwarz inequality";
+and a punctuation dash (also called an "em-dash") in place of a comma, semicolon,
+colon or parentheses---like this.
+
+Generating an ellipsis \ldots\ with the right spacing
+around the periods requires a special command.
+
+## Citation
+
+Simple author and year cite: \cite{billingsley2013convergence}.
+Multiple bibliography items cite: \cite{billingsley2013convergence,bourbaki1966general}.
+Author only cite: \citeauthor{ethier1985markov}.
+Year only cite: \citeyear{prokhorov1956convergence}.
+
+# Fonts
+
+Please use text fonts in text mode, e.g.:
+\begin{itemize}
+\item[]\textrm{Roman}
+\item[]\textit{Italic}
+\item[]\textbf{Bold}
+\item[]\textsc{Small Caps}
+\item[]\textsf{Sans serif}
+\item[]\texttt{Typewriter}
+\end{itemize}
+Please use mathematical fonts in mathematical mode, e.g.:
+\begin{itemize}
+\item[] $\mathrm{ABCabc123}$
+\item[] $\mathit{ABCabc123}$
+\item[] $\mathbf{ABCabc123}$
+\item[] $\boldsymbol{ABCabc123\alpha\beta\gamma}$
+\item[] $\mathcal{ABC}$
+\item[] $\mathbb{ABC}$
+\item[] $\mathsf{ABCabc123}$
+\item[] $\mathtt{ABCabc123}$
+\item[] $\mathfrak{ABCabc123}$
+\end{itemize}
+Note that \verb|\mathcal, \mathbb| belongs to capital letters-only font typefaces.
+
+# Notes
+
+Footnotes[^1]
+pose no problem.\footnote{Note that footnote number is after punctuation.}
+
+[^1]: This is an example of a footnote.
+
+# Quotations
+
+Text is displayed by indenting it from the left margin. There are short quotations
+
+> This is a short quotation.  It consists of a
+> single paragraph of text.  There is no paragraph
+> indentation.
+
+and longer ones.
+\begin{quotation}
+This is a longer quotation.  It consists of two paragraphs
+of text.  The beginning of each paragraph is indicated
+by an extra indentation.
+
+This is the second paragraph of the quotation.  It is just
+as dull as the first paragraph.
+\end{quotation}
+
+# Environments
+
+\subsection{Examples for \emph{\texttt{plain}}-style environments}
+\begin{axiom}\label{ax1}
+This is the body of Axiom \ref{ax1}.
+\end{axiom}
```

Comment:
Thanks for the tip, I didn't know this Rmarkdown syntax. However, I had a problem. When I try to compile the pdf from the `skeleton.rmd`, it doesn't recognize this syntax. On the other hand, if I use `bookdown::pdf_book` using the `template.tex` it does the trick. The same problem for `# Title {.unnumbered}`.

I'm wondering if I should add something to the `ims_article()` or maybe I'm missing something here.

An additional question: How can I translate to this syntax this chunk of code:

```latex
\begin{proof}[Proof of Lemma \ref{le1}]
This is the body of the proof of Lemma \ref{le1}.
\end{proof}
```

and this title section

```latex
\subsection{First subsection of Appendix \protect\ref{appB}}
```

I updated Rmardown to the one in GitHub. Some additional info, just in case:

```r
sessioninfo::session_info()
- Session info --------------------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 4.0.3 (2020-10-10)
 os       Windows 10 x64              
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  German_Switzerland.1252     
 ctype    German_Switzerland.1252     
 tz       Europe/Berlin               
 date     2021-02-10                  

- Packages ------------------------------------------------------------------------------------------------------------------
 ! package     * version date       lib source                            
   assertthat    0.2.1   2019-03-21 [1] CRAN (R 4.0.2)                    
   callr         3.5.1   2020-10-13 [1] CRAN (R 4.0.3)                    
   cli           2.3.0   2021-01-31 [1] CRAN (R 4.0.3)                    
   crayon        1.4.1   2021-02-08 [1] CRAN (R 4.0.3)                    
   curl          4.3     2019-12-02 [1] CRAN (R 4.0.2)                    
   digest        0.6.27  2020-10-24 [1] CRAN (R 4.0.3)                    
   evaluate      0.14    2019-05-28 [1] CRAN (R 4.0.2)                    
   fortunes      1.5-4   2016-12-29 [1] CRAN (R 4.0.0)                    
   glue          1.4.2   2020-08-27 [1] CRAN (R 4.0.2)                    
   htmltools     0.5.1.1 2021-01-22 [1] CRAN (R 4.0.3)                    
   knitr         1.31    2021-01-27 [1] CRAN (R 4.0.3)                    
   pkgbuild      1.2.0   2020-12-15 [1] CRAN (R 4.0.3)                    
   prettyunits   1.1.1   2020-01-24 [1] CRAN (R 4.0.2)                    
   processx      3.4.5   2020-11-30 [1] CRAN (R 4.0.3)                    
   ps            1.5.0   2020-12-05 [1] CRAN (R 4.0.3)                    
   R6            2.5.0   2020-10-28 [1] CRAN (R 4.0.3)                    
   remotes       2.2.0   2020-07-21 [1] CRAN (R 4.0.2)                    
   rlang         0.4.10  2020-12-30 [1] CRAN (R 4.0.3)                    
 V rmarkdown     2.6     2021-02-10 [1] Github (rstudio/rmarkdown@a62cb20)
   rprojroot     2.0.2   2020-11-15 [1] CRAN (R 4.0.3)                    
   rstudioapi    0.13    2020-11-12 [1] CRAN (R 4.0.3)                    
   rticles     * 0.18.3  2021-02-10 [1] local                             
   sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 4.0.2)                    
   withr         2.4.1   2021-01-26 [1] CRAN (R 4.0.3)                    
   xfun          0.20    2021-01-06 [1] CRAN (R 4.0.3)                    
   yaml          2.2.1   2020-02-01 [1] CRAN (R 4.0.2)                    

[1] C:/Users/ualvaro/AppData/Local/Programs/R-4.0.3/library

 V -- Loaded and on-disk version mismatch.
```

Reply:
Thanks a lot for the tip. I didn't know this syntax. I changed most of them to this new syntax. I also added a comment pointing out that this syntax only works with `Rmarkdown >= 2.7`, I hope this is not a problem for those using the CRAN version.